infrastructures/infrastructure-azure/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AzureInfrastructure.java

```diff
@@ -71,43 +73,43 @@
 
     private final static int GRAPH_ENDPOINT_INDEX = 7;
 
-    private final static int RM_HOSTNAME_INDEX = 8;
+    private final static int RM_HTTP_URL_INDEX = 8;
 
     private final static int CONNECTOR_IAAS_URL_INDEX = 9;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)